src/views/BasicPage.vue

```diff
@@ -14,14 +14,51 @@
 				required: true
 			}
 		},
+		data() {
+			return {
+				paddingLeft: 0, 
+			}
+		},
 		components: {
 			'markdown-interpreter': MarkdownInterpreter,
+		},
+
+		created() {						
+			if(!this.isMobile()){
+				this.paddingLeft = 200;
+			}
+		},
+		mounted() {			
```

Comment:
extra white line and removing excess spaces after `{`
```suggestion
		},
		
		mounted() {
```